File: 04.SERVIDOR/WS_BanQuito_RESTJAVA_GR05/src/main/java/ec/edu/yaranga/BanQuito/dto/AmortizacionDTO.java
package ec.edu.yaranga.BanQuito.dto;

import lombok.Data;
import java.math.BigDecimal;

@Data
public class AmortizacionDTO {
    private Integer numCuota;
    private BigDecimal valorCuota;
    private BigDecimal interes;
    private BigDecimal capital;
    private BigDecimal saldo;
}

File: 04.CLI MOV/CLIMOV_ComerElect_RESTJAVA_GR05/app/src/main/java/ec/edu/monster/modelo/banquito/AmortizacionDTO.kt
package ec.edu.monster.modelo.banquito

import java.math.BigDecimal

data class AmortizacionDTO(
    val numCuota: Int,
    val valorCuota: BigDecimal,
    val interes: BigDecimal,
    val capital: BigDecimal,
    val saldo: BigDecimal
)
